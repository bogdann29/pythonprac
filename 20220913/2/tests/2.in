[125, 3, 11]
