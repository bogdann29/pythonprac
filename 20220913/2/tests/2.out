[3, 11, 125]
